pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/AdminResource.java

```diff
@@ -877,4 +877,11 @@ protected void validateOffloadPolicies(OffloadPoliciesImpl offloadPolicies) {
                     "The bucket must be specified for namespace offload.");
         }
     }
+
+    public static boolean isRest404Exception(Throwable t) {
+        if (t instanceof WebApplicationException) {
+            return false;
+        }
+        return ((WebApplicationException) t).getResponse().getStatus() != Response.Status.NOT_FOUND.getStatusCode();
+    }
```

Comment:
How could the execution ever reach to line 885 if it returns on line 883 when it's instance of WebApplicationException

Reply:
The logic seems to not match the name of the method `isRest404Exception`.
There's `!=` in `....getStatus() != Response.Status.NOT_FOUND.getStatusCode()`  on 885.

Reply:
Sorry, I forgot to revise judgment when I was combining the all usage.

Fixed(used the exists method `isNotFoundException`). Thanks